bin/node/runtime/src/weights/pallet_balances.rs

```diff
@@ -0,0 +1,29 @@
+use frame_support::weights::{Weight, constants::RocksDbWeight as DbWeight};
```

Comment:
Needs license header.

Reply:
Will add here, but need to think how we would add this in our automation pipeline.

Reply:
Is the complete file being automatically generated? Perhaps we can add a preamble to the generator that always adds a static header (and also maybe a note `//! THIS FILE WAS AUTOMATICALLY GENERATED`)?